src/__test__/e2e.test.ts

```diff
@@ -29,21 +29,27 @@ test(
     await fs.writeFile(
       "./prisma/schema.prisma",
       `datasource db {
-        provider = "sqlite"
-        url      = "file:./dev.db"
+        provider = "postgresql"
+        url      = "env(DATABASE_URL)"
```

Comment:
I think this shouldn't actually work in CI since we're not running a PG server there. Either we skip this in the e2e test or we need to set up a PG server to test against. The server URL should be hard-coded in the test as well so it's deterministic.

Reply:
Got it - the reason I changed it is because enum arrays are not supported in SQLite so prisma wouldn't generate the schema during the test run. Will have more of a think about this. Perhaps skipping it would be ok